src/shipping-zones/shipping-zones.controller.ts

```diff
@@ -0,0 +1,45 @@
+import { 
+    Controller,
+    Get,
+    Post,
+    Put,
+    Delete,
+    Body,
+    Param,
+} from '@nestjs/common';
+import { ApiTags } from '@nestjs/swagger';
+import { ShippingZonesService } from './shipping-zones.service';
+import { CreateShippingZoneDto } from './dto/create-shipping-zone.dto';
+import { UpdateShippingZoneDto } from './dto/update-shipping-zone.dto';
+
+@ApiTags('Shipping-zones')
```

Comment:
Change this to "Shipping Zones" for consistency with the other api tags in the documentation.